modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/consensus/ConsensusStorage.scala

```diff
@@ -353,6 +399,51 @@ object ConsensusStorage {
             }
           }
 
```

Comment:
Function below can be used for event log tracing at much better granularity than grafana. It does emit a lot of log noise but it's the main information we need.